package-lock.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "bybit-api",
-  "version": "3.10.10",
+  "version": "3.10.11",
   "lockfileVersion": 2,
   "requires": true,
   "packages": {
     "": {
       "name": "bybit-api",
-      "version": "3.10.10",
+      "version": "3.10.11",
```

Comment:
Can you run `npm install` and commit the updated package-lock? It's still referring to the previous version (must be that you ran npm install before updating the version in package.json